packages/plus/src/components/Navigation/components/Item.tsx

```diff
@@ -805,7 +805,7 @@ export const Item = memo(
             flex={1}
             width="100%"
           >
-            <WrapText as="span" variant="bodySmall">
+            <WrapText as="span" variant="bodySmall" whiteSpace="pre-wrap">
```

Comment:
Here it's also incorrect, the condition is `white-space: ${({ animation }) => (animation ? 'nowrap' : 'normal')};` so if animation doesn't exists, which is the case here, it should be normal not pre-wrap